src/Processor/FileProcessor.php

```diff
@@ -38,7 +38,7 @@ public function __invoke(string|null $fileName): \Generator
             $path = $this->directory.'/'.$fileName;
 
             $this->filesystem->dumpFile($path, $resource);
-            $this->logger?->debug('{processor}: content dumped to "{file}".', ['processor' => self::class, 'file' => $fileName]);
+            $this->logger?->debug('{processor}: content dumped to "{file}" on {path}.', ['processor' => self::class, 'file' => $fileName, 'path' => $path]);
```

Comment:
What do you think of :
```suggestion
            $this->logger?->debug('{processor}: content dumped to "{path}".', ['processor' => self::class, 'path' => $path]);
```
?

Reply:
yes both suit me, my main goal is to add the full path, not just the filename alone

Reply:
:+1: For this version.